webapp/ansible/roles/celery/tasks/install.yml

```diff
@@ -1,10 +1,4 @@
 ---
-
-- name: Install python-pip.
-  apt: name=python-pip install_recommends=no
-  tags:
-      - celery-install
-
 - name: Install Celery.
   command: pip install celery
```

Comment:
@gouzouni625 maybe this is irrelevant with your commit but if not can you change this to use the pip module instead of command?